### ISSUE TITLE
About training code for teacher model

### ISSUE THREAD

**Issue by Qianqian3764** [Sungmin-Woo/ProDepth#19]
Hi, thanks for your nice work!

Could you share the training process and code for teacher model (single network).  I notice that its training loss is different from typical methods!

![Image](https://github.com/user-attachments/assets/ba562f42-dc21-4535-9f14-26c422634b62)
